cert/vault_client.go

```diff
@@ -132,3 +137,14 @@ func (c *vaultClient) keepTokenAlive() {
 		timer.Reset(ttl / 2)
 	}
 }
+
+func getVaultTokenFromFile(c string) string {
+	b, err := ioutil.ReadFile(c) // just pass the file name
+	if err != nil {
+		log.Printf("[WARN] vault: Failed to get Token From File: %s", err)
+	}
+	str := string(b)
+	str = strings.TrimSuffix(str, "\n")
```

Comment:
Any reason to not call `strings.TrimSpace` here? Some config management tools are finicky about whitespace in their template engines, for instance. Copy-pasting can also easily add whitespace by accident.

Reply:
I'll update to TrimSpace.